src/backend/packages/functions/src/appointment.ts

```diff
@@ -0,0 +1,83 @@
+import { APIGatewayProxyHandler, Context } from 'aws-lambda'
+import { APIGatewayProxyEvent } from 'aws-lambda'
+
+function getPathWithoutParams(path: string, pathParameters: { [name: string]: string } | null) {
```

Comment:
The pathParams in SST is also weird.. is there a better way of doing this?

Reply:
sst suggests using middy but its kinda poopy for typescript - I'll see what i do in my other project